src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import original from 'react95/dist/themes/original';
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import {
+  styleReset,
+  Button,
+  Window,
+  WindowHeader,
+  WindowContent,
+} from 'react95';
+
+import { createGlobalStyle, ThemeProvider } from 'styled-components';
+
+/* Pick a theme of your choice */
+
+import { TodoList } from './components/TodoList';
+
+// api
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+// types
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+const GlobalStyles = createGlobalStyle`
+  ${styleReset}
+  @font-face {
+    font-family: 'ms_sans_serif';
+    font-weight: 400;
+    font-style: normal
+  }
+  @font-face {
+    font-family: 'ms_sans_serif';
+    font-weight: bold;
+    font-style: normal
+  }
+  body, input, select, textarea {
+    font-family: 'ms_sans_serif';
+  }
+`;
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find((user) => user.id === userId)
+  || null;
+}
+
+const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [newTodos, setNewTodos] = useState<Todo[]>(todos);
+  const [newUserId, setNewUserId] = useState(0);
+  const [newTodoName, setNewTodoName] = useState('');
+  const [hasUserError, setHasUserError] = useState(false);
+  const [hasTodoNameError, setHasTodoNameError] = useState(false);
+
+  const handleReset = () => {
+    setNewUserId(0);
+    setNewTodoName('');
+  };
+
+  const addNewTodo = (title: string, userId: number) => {
+    const newTodo: Todo = {
+      id: newTodos.length + 1,
+      title,
+      completed: false,
+      userId,
+      user: getUserById(userId),
+    };
+
+    setNewTodos((previousGoods) => [...previousGoods, newTodo]);
+  };
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (newTodoName && newUserId) {
```

Comment:
Consider using optional chaining to avoid null pointer exceptions. For example, `newTodo?.user?.name` instead of `newTodo.user.name`.